tests/vm_test.go

```diff
@@ -575,7 +573,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			}
 		})
 
-		It("[test_id:7679]should report an error status when data volume error occurs", func() {
+		It("[test_id:7679]should report an error status when data volume error occurs", decorators.Conformance, func() {
```

Comment:
We should not use `Skip` inside Conformance tests

Reply:
Removed from the conformance set